Source/Core/DolphinQt2/resource.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by DolphinWX.rc
```

Comment:
This should likely reference DolphinQt instead.

Reply:
Whoops, you're right.